packages/www/components/Dashboard/StreamDetails/Delete.tsx

```diff
@@ -33,57 +35,63 @@ const Delete = ({ stream, invalidate, ...props }) => {
         color="red">
         Delete
       </Box>
+      <Portal>
+        <Overlay />
+        <AlertDialogContent
+          css={{ maxWidth: 450, px: "$5", pt: "$4", pb: "$4" }}>
+          <AlertDialogTitle asChild>
+            <Heading size="1">Delete stream?</Heading>
+          </AlertDialogTitle>
+          <AlertDialogDescription asChild>
+            <Text
+              size="3"
+              variant="neutral"
+              css={{ mt: "$2", lineHeight: "22px" }}>
+              Are you sure you want to delete stream {stream.name}? Deleting a
```

Comment:
I haven't gone through and actually tested this in the UI, but it seems like this interpolation wouldn't work on first glance (and I think this existed before this PR)

Reply:
Nevermind, looking at this again and it's all good!